### ISSUE TITLE
Activity Bar doesn't render tasks with "\t" as indentation

### ISSUE THREAD

**Issue by peter-mccrory** [fabiospampinato/vscode-todo-plus#114]
When I specify the user setting `"todo.indentation": "\t"` my activity bar no longer renders tasks correctly. I see the names of the projects but the tasks are not shown.

For example, if my TODO file reads (with the indentation as a tab instead of spaces)
```
General: (1)
    - This is a task

Second Project: (1)
    - A second task
```
the activity bar will read
```
General:
Second Project:
```
Unlike in the default case, there is no tree navigation characters and the tasks are not shown.

**Comment by peter-mccrory** [comment]
Thanks for the quick response!

These are my user settings for TODO+
```
{
	"todo.colors.done": "#2b7a00",
	"todo.colors.tag": "#7100bd",
	"todo.colors.project": "#ff3f0f",
	"[todo]": {
		"editor.wordWrap": "bounded",
		"editor.wordWrapColumn": 72
	},
	"todo.symbols.box": "-",
	"todo.indentation": "\t"
}
```
Here is a file that replicates the issue.
[todolist.txt](https://github.com/fabiospampinato/vscode-todo-plus/files/2325016/todolist.txt)

And here is a screenshot of the activity bar and the file (`TODO.taskpaper` has similar issues as the test case that I've uploaded).
<img width="537" alt="screenshot_activity_bar_issue" src="https://user-images.githubusercontent.com/7097207/44676574-8d202100-a9e8-11e8-8f31-4f278b4e63b7.png">

Finally, if I change back to the default setting, `"todo.indentation": "  "`, and update the task list to have two-space indentations, the issue goes away.

Thanks!

**Comment by fabiospampinato** [comment]
Should be fixed in v4.4.3. Thanks a lot for reporting the issue!

**Comment by peter-mccrory** [comment]
Thanks!
